product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx

```diff
@@ -0,0 +1,213 @@
+
+## Set up the repository
+
+1. To create the repository configuration file, assume superuser privileges, and invoke the following command:
+    
+    ```shell
+    dnf -y install https://yum.enterprisedb.com/edbrepos/edb-repo-latest.noarch.rpm
+    ```
+    
+    In the directory `/etc/yum.repos.d`, the repository configuration file `edb.repo` is created, which a text file is containing a list of EnterpriseDB repositories, each denoted by an entry starting with the text `[repository_name]`.
+
+2. Enable access to the packages in any of these repositories:
+
+    - Using your requested credentials for the EDB Yum Repository, substitute the user name and password for the `<username>:<password>` placeholders of the baseurl parameter.
+    - Change the setting of the enabled parameter to `enabled=1`.
+  
+  For example, to access the server `libs` package from the repository for Advanced Server version 14, enable the following entry:
+
+  ```shell
+  [edbas96]
+  name=EnterpriseDB Advanced Server 14 $releasever - $basearch
+  baseurl=http://<username>:<password>@yum.enterprisedb.com/14/redhat/rhel-$releasever-$basearch
+  enabled=1
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/ENTERPRISEDB-GPG-KEY
+  ```
+
+  When enabling one version, be sure the other entries are disabled (that is, the parameter setting is `enabled=0` for the non-selected entries). 
+    
+  For example, to access Replication Server version 7.0, enable the following entry:
+
+  ```shell
+  [enterprisedb-xdb70]
+  name=EnterpriseDB XDB 7.0 $releasever - $basearch
+  baseurl=http://<username>:<password>@yum.enterprisedb.com/xdb70/redhat/rhel-$releasever-$basearch
+  enabled=0
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/ENTERPRISEDB-GPG-KEY
+  ```
+
+## Install the package
+
+You can install all Replication Server components with a single install command, or you may choose to install selected, individual components by installing only those particular packages.
+
+To install all Replication Server components:
+
+```shell
+dnf -y install edb-xdb
+```
+
+To install an individual component:
+
+```shell
+dnf install package_name
+```
+
+Where `package_name` is:
+
+| Package name         | Component                                                                  |
+| -------------------- | -------------------------------------------------------------------------- |
+| `edb-xdb-console`    | EPRS Replication Console and the Replication Server Command Line Interface |
+| `edb-xdb-publisher`  | Publication server                                                         |
+| `edb-xdb-subscriber` | Subscription server                                                        |
+
+The following is an example:
+
+```shell
```

Comment:
FWIW: you can separate the output from the command by including `__OUTPUT__` on a line by itself